charts/mageai/templates/webservice.yaml

```diff
@@ -79,7 +79,7 @@ spec:
             timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
           {{- end }}
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.webServer.resources | nindent 12 }}
```

Comment:
can you make it backward compatible?

Reply:
i cant understand why we required backward compatibility. While installation its gonna use the default resource configuration.

Reply:
Currently, user configures the web server and scheduler resources using the resources configuration in values.yaml
If they upgrade the helm chart version, their current configuration will not be applied anymore.

So in your change, you should fallback to the `.Values.resources` if `.Values.webServer.resources` is not present.

Reply:
@wangxiaoyou1993 fallback support applied